redox/processor_neworder.go

```diff
@@ -4,13 +4,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	t "github.com/tidepool-org/clinic-worker/types"
 	"net/http"
 	"net/mail"
 	"strings"
 	"time"
 
-	"github.com/oapi-codegen/runtime/types"
+	codegentypes "github.com/oapi-codegen/runtime/types"
```

Comment:
Ironically, this was all for naught, as with your naming of that anonymous struct, the need for "github.com/tidepool/clinic-worker/types" goes away. With that gone, this import doesn't need to be renamed. 

Maybe a fix for next time.